ui/src/app/images/components/ImagesTable/ImagesTable.tsx

```diff
@@ -64,14 +71,26 @@ const generateImageRow = (image: ImageValue) => {
 /**
  * Generates a row based on a resource.
  * @param resource - the resource from which to generate the row.
- * @param toDelete - whether the resource is slated for deletion.
+ * @param commissioningRelease - the name of the default commissioning release.
+ * @param expanded - the resource id of the expanded row.
+ * @param setExpanded - function to expand the row of a resource.
+ * @param unchecked - whether the resource checkbox is unchecked.
  * @returns row generated from resource.
  */
-const generateResourceRow = (resource: BootResource, toDelete = false) => {
+const generateResourceRow = (
+  resource: BootResource,
+  commissioningRelease: string | null,
+  expanded: BootResource["id"] | null,
+  setExpanded: (id: BootResource["id"] | null) => void,
+  unchecked = false
+) => {
+  const { os, release } = splitResourceName(resource.name);
+  const canBeDeleted = !(os === "ubuntu" && release === commissioningRelease);
+  const isExpanded = expanded === resource.id;
```

Comment:
The row also needs to add the active class when it is expanded to show the white background:
```typescript
className: classNames("p-table__row", {
        "is-active": isExpanded,
      }),
```